src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -677,6 +677,31 @@ def get_printable_command(command, fuzzer_path, use_minijail):
   return engine_common.get_command_quoted(command)
 
 
+def maybe_use_mutator_plugin(target_name, extra_env, fuzzing_strategies,
+                             chroot):
+  """Decide whether to use a mutator plugin. If yes and there is a usable plugin
+  available for |target_name|, then add it to LD_PRELOAD in |extra_env|, add a
+  strategy to |fuzzing_strategies|, and add chroot bindings if |chroot| is not
+  None."""
+  if not do_mutator_plugin():
+    return
+
+  mutator_plugin_path = mutator_plugin.get_mutator_plugin(target_name)
+  if not mutator_plugin_path:
+    return
+
+  logs.log('Using mutator plugin: %s' % mutator_plugin_path)
+  # TODO(metzman): Change the strategy to record which plugin was used, and
+  # not simply that a plugin was used.
+  extra_env['LD_PRELOAD'] = mutator_plugin_path
+  fuzzing_strategies.append(strategy.MUTATOR_PLUGIN_STRATEGY)
```

Comment:
move this line 697 in caller, similar to comment above. that way, also you dont need to pass fuzzing_strategies

Reply:
Done.